app/upgrades/v4/constants.go

```diff
@@ -0,0 +1,25 @@
+package v4
+
+import (
+	storetypes "cosmossdk.io/store/types"
+	"github.com/quasar-finance/quasar/app/upgrades"
+	marketmaptypes "github.com/skip-mev/slinky/x/marketmap/types"
+	oracletypes "github.com/skip-mev/slinky/x/oracle/types"
+)
+
+const (
+	UpgradeName                = "v4"
+	MarketMapAuthorityMultisig = "" //TODO: add
```

Comment:
todo??

Reply:
this needs to be setup when we are ready to update